openjdk8/src/test/resources/structure.yaml

```diff
@@ -8,6 +8,11 @@ commandTests:
 - name: 'correct java version is installed'
   command: ['java', '-version']
   expectedError: ['openjdk version "1\.${openjdk.version.major}.*${openjdk.version.minor}"']
+- name: 'ensure java7 is not installed as a dependency accidentally'
+  setup: [[ 'chmod', '+x', '/workspace/openjdk8/src/test/workspace/test-no-java-7.bash' ]]
+  command: [ '/workspace/openjdk8/src/test/workspace/test-no-java-7.bash' ]
+  expectedError: ['WARNING.*apt does not have a stable CLI interface yet.']
+  expectedOutput: ['OK.* is not installed']
```

Comment:
I'm not sure that this will catch if java 7 is installed. I believe that the test will pass if the `expectedOutput` or `expectedError` are matched at least one time.